logstash-core/src/main/java/org/logstash/instrument/monitors/ProcessMonitor.java

```diff
@@ -90,6 +93,33 @@ private static short scaleLoadToPercent(double load) {
                 return -1;
             }
         }
+
+        // The method `getSystemCpuLoad` is deprecated in favour of `getCpuLoad` since JDK14
+        // This method uses reflection to use the correct method depending on the version of
+        // the JDK being used.
+        private short getSystemCpuLoad(){
+            if (CPU_LOAD_METHOD == null){
+                return -1;
+            }
+            try {
+                return scaleLoadToPercent((double)CPU_LOAD_METHOD.invoke(osMxBean));
+            } catch (Exception e){
+                return -1;
+            }
+        }
+    }
+
+    /**
+     * Retrieve the correct name of the method to get CPU load.
+     * @return Method if the method could be found, null otherwise
+     */
+    private static Method getCpuLoadMethod(){
+        try{
+            String methodName = (LogstashJavaCompat.isJavaAtLeast(14)) ? "getCpuLoad" : "getSystemCpuLoad";
+            return Class.forName("com.sun.management.OperatingSystemMXBean").getMethod(methodName);
+        } catch (ReflectiveOperationException e){
```

Comment:
If we get here probably it's because the `com.sun.management` is not present, perhaps because it's not running on OpenJDK derived runtime, probably we should log any warning message in this case, WDYT?